cdc/sink/metrics/statistics.go

```diff
@@ -107,13 +109,22 @@ type Statistics struct {
 	metricExecDDLHis   prometheus.Observer
 	metricExecBatchHis prometheus.Observer
 	metricExecErrCnt   prometheus.Counter
+
+	metricRowSizesHis prometheus.Observer
 }
 
 // AddRowsCount records total number of rows needs to flush
 func (b *Statistics) AddRowsCount(count int) {
 	atomic.AddUint64(&b.totalRows, uint64(count))
 }
 
+// ObserveRows record the size of all received `RowChangedEvent`
+func (b *Statistics) ObserveRows(rows ...*model.RowChangedEvent) {
+	for _, row := range rows {
+		b.metricRowSizesHis.Observe(float64(row.ApproximateDataSize))
```

Comment:
Could you record large rows only? Histogram.Observe overhead is large when there are many rows.

Reply:
done, set `Bucket` start from 1MB.